# Message that contains a trajectory for mobile robot navigation

std_msgs/Header header
geometry_msgs/Pose[] pose_seq


